server/neptune/workflows/internal/deploy/terraform/state.go

```diff
@@ -52,40 +52,6 @@ func (n *StateReceiver) Receive(ctx workflow.Context, c workflow.ReceiveChannel,
 			workflow.GetLogger(ctx).Error(errors.Wrap(err, "auditing apply job event").Error())
 		}
 	}
```

Comment:
The sns notifier should replace  the call to `emitApplyEvents` right?

Reply:
wow not sure how i even left this in, removed.